bond/commands/select.py

```diff
@@ -25,6 +32,11 @@ def run(self, args):
             if args.port:
                 bond.database.set_bond(args.BONDID, "port", args.port)
                 LogLine("Set %s port %s" % (args.BONDID, args.ip))
+            threads = get_async(args.BONDID,
```

Comment:
Line break on the first argument as well.

Reply:
Good catch, I just `black`ified the changes. I'd like some help with automating that when you or @apastolibra are back from your break.